api/v4/source/definitions.yaml

```diff
@@ -4407,12 +4425,18 @@ components:
         after:
           type: string
           description: The ID of the user to start the test after (for pagination).
+        channelId:
+          type: string
+          description: The channel ID to contextually test the expression against (required for channel admins).
     CELExpression:
       type: object
       properties:
         expression:
           type: string
           description: The CEL expression to visualize.
+        channelId:
+          type: string
+          description: The channel ID to contextually test the expression against (required for channel admins).
```

Comment:
Some of these were picked up as not documented. If we want to keep it that way we can remove them.